addons/sourcemod/translations/showplayerclips.phrases.txt

```diff
@@ -1,12 +1,21 @@
 "Phrases"
 {
+	// Комментарий переводчика:
+	// Перевел фразу "Player clips" как "плагин", 
+	// так как не думаю что нет смысла повторяться мол "этот же плагин и включен"
+	// Я думаю ( думать вредно (с) Феникс ) по префиксу (которого нет) и так ясно какой плагин включается

```

Comment:
And that comment is unnecessary here, as it's main translation file and not a specific for one language.